elements/lisk-transactions/src/11_vote_transaction.ts

```diff
@@ -1,269 +0,0 @@
-/*
```

Comment:
I think it would be better for the new vote transaction to take this number to avoid having a gap in the transaction numbers.

Reply:
yes I think we need to number them, but I was thinking since we have several new transaction coming in we can rename at the end?